recovery_backend/backend/configmap.go

```diff
@@ -12,9 +12,14 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 
 	"github.com/longhorn/longhorn-manager/datastore"
+	longhorn "github.com/longhorn/longhorn-manager/k8s/pkg/apis/longhorn/v1beta2"
 	"github.com/longhorn/longhorn-manager/types"
 )
 
+var (
+	longhornFinalizerKey = longhorn.SchemeGroupVersion.Group
+)
```

Comment:
WIll this be used? didn't see any reference in configmap.go

Reply:
Removed. Thank you.